EnglishFilter/sections/css_extended.txt

```diff
@@ -3,6 +3,9 @@
 !
 ! https://github.com/AdguardTeam/AdguardFilters/issues/59035
 comicastle.org#?#.card > .card-body.text-center:has(> .adsbygoogle)
+! https://github.com/AdguardTeam/AdguardFilters/issues/59032
+file-upload.com#?#.container > div.page-wrap > div.text-center[style^="margin-bottom: 20px;"]:contains(ads)
+file-upload.com#?#.container > div.page-wrap > center:contains(ads) 
```

Comment:
I don't see these elements.

Reply:
you can see in on last page before download
<details>
<summary>Screenshot</summary>
<img src="https://user-images.githubusercontent.com/18536328/87535916-e3f87780-c6a0-11ea-8b37-67e13c2eb473.png">
</details>